includes/components/class-admin.php

```diff
@@ -946,6 +949,312 @@ protected function get_extensions_tab() {
+
+					if ( file_exists( $file ) ) {
+						$headers = array_filter(
+							get_file_data(
+								$file,
+								[
+									'name'    => 'Plugin Name',
+									'version' => 'Version',
+									'url'     => 'Plugin URI',
+									'author'  => 'Author',
+								]
+							)
+						);
+
+						$value = [
+							'label' => hp\get_array_value( $headers, 'name' ),
+							'value' => sprintf( esc_html__( 'by %1$s - %2$s', 'hivepress' ), hp\get_array_value( $headers, 'author' ), hp\get_array_value( $headers, 'version' ) ),
+							'url'   => hp\get_array_value( $headers, 'url' ),
+						];
+
+						return $value;
+					}
+				},
+				(array) get_option( 'active_plugins' )
+			)
+		);
+
+		// Get theme info.
+		$theme = wp_get_theme( get_template() );
+
+		$theme_info = [
+			[
+				'label'    => esc_html__( 'Name' ),
+				'label_en' => 'Name',
+				'value'    => $theme->get( 'Name' ),
+			],
+			[
+				'label'    => esc_html__( 'Version' ),
+				'label_en' => 'Version',
+				'value'    => $theme->get( 'Version' ),
+			],
+			[
+				'label'    => esc_html__( 'Author URL', 'hivepress' ),
+				'label_en' => 'Author URL',
+				'value'    => $theme->get( 'AuthorURI' ),
+			],
+			[
+				'label'    => esc_html__( 'Child theme', 'hivepress' ),
+				'label_en' => 'Child theme',
+				'value'    => is_child_theme() ? '+' : '-',
+			],
+		];
+
+		if ( is_child_theme() ) {
+			$parent_theme = wp_get_theme( $theme->template );
+
+			$theme_info = array_merge(
+				$theme_info,
+				[
+					[
+						'label'    => esc_html__( 'Parent Theme Name', 'hivepress' ),
+						'label_en' => 'Parent Theme Name',
+						'value'    => $parent_theme->name,
+					],
+					[
+						'label'    => esc_html__( 'Parent Theme Version', 'hivepress' ),
+						'label_en' => 'Parent Theme Version',
+						'value'    => $parent_theme->version,
+					],
+				]
+			);
+		}
+
+		$theme_info[] = [
+			'label'    => esc_html__( 'HivePress support', 'hivepress' ),
+			'label_en' => 'HivePress support',
+			'value'    => current_theme_supports( 'hivepress' ) ? '+' : '-',
+		];
+
+		// Check GET request
+		$is_remote_get = false;
+
+		// Use wp_remote_get to send the GET request
+		$response_get = wp_remote_get( 'https://api.wordpress.org/core/version-check/1.7/' );
+
+		// Check for any errors
+		if ( ! is_wp_error( $response_get ) && $response_get['response']['code'] >= 200 && $response_get['response']['code'] < 300 ) {
+			$is_remote_get = true;
+		}
+
+		// Check GET request
+		$is_remote_post = false;
+
+		$response_post = wp_remote_post( 'https://api.wordpress.org/core/version-check/1.7/' );
+
+		// Check for any errors
+		if ( ! is_wp_error( $response_post ) && $response_post['response']['code'] >= 200 && $response_post['response']['code'] < 300 ) {
+			$is_remote_post = true;
+		}
+
+		// Get report.
+		$report = [
+			'content'  => '',
+			'sections' => [
+				[
+					'label'    => esc_html__( 'WordPress enviroment', 'hivepress' ),
+					'label_en' => 'WordPress enviroment',
```

Comment:
Let me know why duplicate params label_en are required. Also please check the comment above regarding brands in the translatable strings, make sure that the number of newly added custom translatable strings is optimized

Reply:
Unfortunately, the solution to disable gettext feature does not work so `label_en` parameter is needed